components/GiveStarCard.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import Modal from 'react-bootstrap/Modal'
+import Button from 'react-bootstrap/Button'
+import InputGroup from 'react-bootstrap/InputGroup'
+import FormControl from 'react-bootstrap/FormControl'
+
+type Props = {
+  show: boolean
+}
+
+const GiveStarCard: React.FC<Props> = props => {
```

Comment:
You should destructure the props to maintain codebase consistency, because the rest of the codebase destructures props.
```
const GiveStarCard: React.FC<Props> = ({ show }}=> {
```

Reply:
Yes will do. :)